calibrators/mapping_based_calibrator/README.md

```diff
@@ -0,0 +1,238 @@
+| `calibration_camera_info_topics`         | `std::vector<string>` |                | The list of camera info topics for `calibration camera`. (currently not used)                                                                                                                 |
+| `calibration_image_topics`               | `std::vector<string>` |                | The list of camera image topics for `calibration camera`. (currently not used)                                                                                                                |
+| `calibration_pointcloud_topics`          | `std::vector<string>` |                | The list of pointcloud topics for `calibration lidars`.                                                                                                                                       |
+| `mapping_lidar_frame`                    | `std::string`         |                | The frame name of the `mapping_lidar`.                                                                                                                                                        |
+| `mapping_registrator`                    | `std::string`         | `NDT` / `GICP` | The type of the PCL registration algorithm used for mapping.                                                                                                                                  |
+| `mapping_verbose`                        | `bool`                | `false`        | Verbose output flag for mapping.                                                                                                                                                              |
+| `use_rosbag`                             | `bool`                | `true`         | Flag to determine if data should be read from a rosbag file.                                                                                                                                  |
+| `mapping_max_frames`                     | `int`                 | `500`          | The maximum number of frames to use for mapping, if the number of frames is larger than this value, the mapper stops and the calibration starts.                                              |
+| `local_map_num_keyframes`                | `int`                 | `15`           | The number of keyframes stored in the local map.                                                                                                                                              |
+| `dense_pointcloud_num_keyframes`         | `int`                 | `10`           | Dense pointcloud for calibration is generated by integrating the pointcloud in the range of [keyframe_id - `dense_pointcloud_num_keyframes`, keyframe_id + `dense_pointcloud_num_keyframes`]. |
```

Comment:
Plural

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/abc53567faad4b11e9043464fb61e7f639f1e604